Xamarin.Forms.Platform.iOS/Controls/Snackbar/SnackbarViews/ActionMessageSnackbarView.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using UIKit;
+using Xamarin.Forms.Platform.iOS.Controls.Snackbar.Extensions;
+
+namespace Xamarin.Forms.Platform.iOS.Controls.Snackbar.SnackbarViews
+{
+    public class ActionMessageSnackbarView : MessageSnackbarView
+    {
+		readonly SnackBar _snackbar;
+
+		public UIButton ActionButton { get; set; }
+
+        public ActionMessageSnackbarView(SnackBar snackbar) : base(snackbar)
+        {
+			_snackbar = snackbar;
```

Comment:
Style is wrong,

Reply:
should I change it to this.snackbar = snackbar?